CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [CHANGE] Distributor: OTLP and push handler replace all non-UTF8 characters with the unicode replacement character `\uFFFD` in error messages before propagating them. #10236
 * [CHANGE] Querier: pass query matchers to queryable `IsApplicable` hook. #10256
+* [CHANGE] Ingester: remove experimental flag `-ingester.log-utilization-based-limiter-cpu-samples`. #10284
```

Comment:
Did we have a CHANGELOG entry when this flag was introduced? If not, I think that this one can also be removed.

Reply:
ah, nice
```suggestion
```